infra/package.json

```diff
@@ -8,7 +8,7 @@
     "test": "jest",
     "cdk": "cdk",
     "setup": "cdk bootstrap --profile=${AWS_PROFILE:=default}",
-    "deploy": "cdk deploy --profile=${AWS_PROFILE:=default} && tool/create_configs.dart",
+    "deploy": "cdk deploy --profile=${AWS_PROFILE:=default} && dart pub get && tool/create_configs.dart",
     "destroy": "cdk destroy --profile=${AWS_PROFILE:=default}"
   },
   "devDependencies": {
```

Comment:
mentioned offline, but please change `ts-node` version below to one that can resolve (from earlier PR).